File: ZAB_FYP_PORTAL FLUTTER FRONTEND/lib/models/area_model.dart
class AreaModel {
  int? iD;
  String? nAME;
  String? dESCRITION;

  AreaModel({this.iD, this.nAME, this.dESCRITION});

  AreaModel.fromJson(Map<String, dynamic> json) {
    iD = json['ID'];
    nAME = json['NAME'];
    dESCRITION = json['DESCRITION'];
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = new Map<String, dynamic>();
    data['ID'] = this.iD;
    data['NAME'] = this.nAME;
    data['DESCRITION'] = this.dESCRITION;
    return data;
  }
}

File: Node-JS-Practice/src/models/area.model.ts
export default class Area {
  public ID: string;
  public NAME: string;
  public DESCRITION: string;
  public INSTITUTEID: string;
  constructor(ID: string, NAME: string, DESCRITION: string,
    INSTITUTEID: string) {
    this.ID=ID;
    this.NAME=NAME;
    this.DESCRITION=DESCRITION;
    this.INSTITUTEID=INSTITUTEID
  }

}
